### DISCUSSION TITLE
Enable "--max-warnings" flag into "next lint" command

### DISCUSSION THREAD

**Discussion by dudusotero** [vercel/next.js#26671] [Ideas]
### Describe the feature you'd like to request

I would like to request [Max Warnings flag](https://eslint.org/docs/user-guide/command-line-interface#-max-warnings) into ```next lint``` command.

The main reason for this is to use together with something like ```lint-staged``` to validade all my files.

<img width="307" alt="image" src="https://user-images.githubusercontent.com/7073199/123555603-19d12780-d75d-11eb-9078-ef1ac8967469.png">
Sometimes I have only warnings but I want stop commit actions like when I have errors.

### Describe the solution you'd like

Add a new flag into "Handling warnings" section, like we already have with ```---quiet``` flag.

<img width="785" alt="image" src="https://user-images.githubusercontent.com/7073199/123555730-eb078100-d75d-11eb-8629-b61f894612dd.png">

### Describe alternatives you've considered

One idea will be just add a ```--ignore-warnings``` flag that will be a boolean value.

But, just use the ```eslint``` way will be better due to a entire community already know how to handle this with pure eslint configuration.

**Comment by housseindjirdeh** [comment]
Added here: #26697 :)

**Reply by sstone2423** [reply]
This doesn't seem to work unfortunately

**Reply by korczas** [reply]
next 14 - not working

**Comment by vicasas** [comment]
@housseindjirdeh This apparently isn't working. I have an application in Next.js that is using `lint-staged` in conjunction with `husky`, when committing I would expect it to fail since there are Eslint errors in the code.

I have the following:

```tsx
const path = require('path')

const buildEslintCommand = (filenames) =>
  `next lint --max-warnings 0 --fix --file ${filenames
    .map((f) => path.relative(process.cwd(), f))
    .join(' --file ')}`

module.exports = {
  '**/*.{js,jsx,ts,tsx}': [buildEslintCommand, 'npm run format:fix'],
  '**/*.{md,mdx,mjs,yml,yaml,css,json}': ['npm run format:fix'],
}

```

**Reply by sstone2423** [reply]
Yeah I'm on next 12.3.4 and this isn't working

**Reply by parandroid42** [reply]
This is still broken in next 14. @housseindjirdeh any chance you could take a look at this again? Thanks!
